app/controllers/api/v2/admin/accounts_controller.rb

```diff
@@ -12,6 +12,7 @@ class Api::V2::Admin::AccountsController < Api::V1::Admin::AccountsController
     ip
     invited_by
     role_ids
+    role_ids: []
```

Comment:
I think adding this to the constant makes sense - originally added in https://github.com/mastodon/mastodon/pull/20625

We may also want to remove the line above which accepts non-array version?